holidays/countries/french_southern_territories.py

```diff
@@ -0,0 +1,49 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from holidays.countries.france import France
+
+
+class HolidaysTF(France):
+    """French Southern Territories holidays.
+
+    Alias of a French subdivision that is also officially assigned
+        its own country code in ISO 31661.
+    See <https://www.iso.org/obp/ui#iso:code:3166:TF>
+    """
+
+    country = "TF"
+    parent_entity = France
+    subdivisions = ()  # Override France subdivisions.
+    subdivisions_aliases = {}  # Override France subdivisions aliases.
+    # This overseas territory was separated in 1955.
+    start_year = 1956
+
```

Comment:
```suggestion

    # Dummy string to avoid generating .mo file error.
    dummy = tr("Holiday")
```
Since our current implementation of .mo files generation (`scripts/l10n/generate_po_files.py`) requires the mandatory presence of localized strings (with `tr()`) when the class has `default_language` attribute, I propose this solution.
@arkid15r, @PPsyrius, what do you think?

Reply:
Now I think it would be better to update `lingva` to allow creating .po files that don't contain msgids.

Reply:
Hey @KJhellico, any update here?!

Reply:
We're going to [update lingva](https://github.com/vacanza/lingva/pull/25) soon (probably today or tomorrow), and it will be possible to create .po files without msgids.

Reply:
Perfect, thanks man!
Ping me here when you do it, please.

Reply:
`lingva` and .po generation script has already been updated.